### DISCUSSION TITLE
I can't find where to enter path settings

### DISCUSSION THREAD

**Discussion by moonshine321** [iterate-ch/cyberduck#15362] [Q&A]
Hi I need to enter my path settings on open connection in order to access my server on Cyberduck but there Is no  button or tab labeled "Advanced" or "More Options." What am I doing wrong or where am I not looking?

**Comment by tatze96** [comment]
Assuming that you are using macOS as the operating system, this is currently expected behavior if you use the *Open Connection* button to open a temporary connection instead of creating a [bookmark](https://docs.cyberduck.io/cyberduck/bookmarks/#add-new-bookmark).
